.circleci/config.yml

```diff
@@ -100,255 +95,215 @@ jobs:
                 exit 1
               fi
           name: "Running tests"
-      -
-        run:
+      - run: 
           command: ~/repo/scripts/notify_failure.sh
           name: "Report failing Master"
           when: on_fail
     working_directory: ~/repo/etl
-  extract_endpoint:
-    docker:
-      -
-        image: "circleci/python:3.6.1"
-      -
-        environment: executable=blob
+  extract_endpoint: 
+    docker: 
+      - image: "circleci/python:3.6.1"
+      - environment: executable=blob
         image: arafato/azurite
-    steps:
-      -
-        checkout:
+    steps: 
+      - checkout: 
```

Comment:
I appreciate the reformatting, but generally it would be better to do that in a separate, prior PR. Makes it easier to identify the critical new parts from the lower-risk reformatting.